Example_DynamicFramework/Demo/AppDelegate.m

```diff
@@ -76,6 +76,16 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     //    StreetHawk.notificationTypes = UIRemoteNotificationTypeAlert;
     //    StreetHawk.isDefaultLocationServiceEnabled = NO;
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(installRegisterSuccessHandler:) name:SHInstallRegistrationSuccessNotification object:nil];
+    
+    SEGAnalyticsConfiguration *configuration = [SEGAnalyticsConfiguration configurationWithWriteKey:@"Z3KDihiA42uXBPr82xULjKA6Fb0p0v38"];
```

Comment:
Where is this key from? Customer's register key in segment?

Reply:
obviously this should be in the sample app codebase not in the library codebase